astropy/units/tests/test_deprecated.py

```diff
@@ -3,11 +3,12 @@
 
 # TEST_UNICODE_LITERALS
 
-"""Regression tests for deprecated units."""
+"""Regression tests for deprecated units or those that are "soft" deprecated
+because they are required for VOUnit support but are not in common use."""
 from __future__ import (absolute_import, unicode_literals, division,
                         print_function)
 
-from .. import deprecated
+from .. import deprecated, required_by_vounit
 from ... import units as u
 from ...tests.helper import pytest  # TODO: Stop using bundled pytest
```

Comment:
Remove this while you're at it? Or is this just because your PRs are based on a relatively old branch?